index.js

```diff
@@ -632,6 +632,54 @@
 
   //. ### Function
 
+  //# curry2 :: ((a, b) -> c) -> a -> b -> c
+  //.
+  //. Curry a binary function.
```

Comment:
I like these short descriptions. For consistency we should use _Curr**ies** a binary function_. Also, we could be more specific about which function gets curried:

_Curries the given binary function._